#!/usr/bin/env python
PACKAGE = "saliency_detector"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()
gen.add("min_target_hue", int_t, 0, "Minimum Hue from Target Description", 120, 0, 180)
gen.add("max_target_hue", int_t, 0, "Maximum Hue from Target Description", 30, 0, 180)
gen.add("min_color_saturation", int_t, 0, "Minimum Saturation to Count Hue", 150, 0, 255)
gen.add("hue_slop", int_t, 0, "Maximum Hue from Target Description", 5, 0, 20)
gen.add("max_inner_outer_hist_correl", double_t, 0, "Maximum correlation between inner and outer region histograms", 0.2, 0.0, 1.0)
gen.add("enable_debug", bool_t, 0, "Draw and publish debug image", False)

exit(gen.generate(PACKAGE, "color_histogram_descriptor", "color_histogram_descriptor_params"))
